boards/arm/beagleconnect_freedom_cc1352r/doc/index.rst

```diff
@@ -0,0 +1,66 @@
+.. _beagleconnect_freedom_cc1352r:
+
+BeagleBoard BeagleConnect Freedom
+#################
+
+Overview
+********
+
+The BeagleBoard BeagleConnect Freedom is a wireless
+IOT board based on the SimpleLink |trade| multi-Standard CC1352R wireless MCU.
+
+
+.. figure:: img/Beagleconnect_Freedom.jpeg
+   :width: 400px
+   :align: center
+   :alt: BeagleBoard BeagleConnect Freedom
+
+   BeagleBoard BeagleConnect Freedom |trade|
+
+Hardware
+********
+
+The BeagleBoard BeagleConnect Freedom IOT board features the CC1352R wireless MCU.
+The BeagleConnect Freedom is the first available BeagleConnect solution consisting
+ of a board and a case which ships programmed and ready to be used.
+ The BeagleConnect Freedom board runs the Zephyr RTOS has mikroBUS ports along
+ with BLE and Sub-GHz radios on it.
+
+The CC13522 wireless MCU has a 48 MHz Arm |reg| Cortex |reg|-M4F SoC and an
```

Comment:
Not "CC13522". Be sure to give credit at the bottom of the article for *who* owns the trademarks, otherwise including the marks isn't useful.